### PR TITLE
Stage reorg

### PR DESCRIPTION
**Pull request by Deckart2**
Hi @awunderground! 

I have made the structure updates to the yaml and other files based on the guidance in [issue 30](https://github.com/awunderground/data-science-for-public-policy2/issues/30) as best I could. I do have a few questions for you (and may have to change some numbers if my assumptions are wrong):

1. Do we want to include the reading list file from the original repo? I assume no
2. Do we want to include the EDA quarto document? I assumed yes
3. Is the plan to combine the course 1 simulation and sampling and course 2 simulation and sampling material? I am assuming no
4. Do we want to include the cloud computing section material from course 1? I am assuming yes
5. Will a new R package section be developed (presumably out of the start of the `functions.qmd` file in this repo? I am assuming yes.

### CODE REVIEW

**Comment by awunderground** [comment]
1. In the long run, yes! For now, no.
2. Yes!
3. No. They are closely related.
4. Not for now.
5. Yes!

**Comment by Deckart2** [comment]
Awesome! I think this section should be ready to go then.

**Review by awunderground — CHANGES_REQUESTED** [review]
This is great!

I sent you the necessary files to add the remaining chapters from the 2nd class on Slack. The chapters may take a long time to render the first time you render the full document. I set up caching, so hopefully this is a rare challenge.

File: 19_predictive-modeling-review.qmd
Comment (awunderground) [inline 19_predictive-modeling-review.qmd:1]: The images and references are broken for this section.

Comment (Deckart2) [reply 19_predictive-modeling-review.qmd:1]: It looks ok over here. Am I mi
Screen Shot 2024-03-17 at 11 16 12 PM
d2ee4-3e1e-431f-8f04-ac89f2fb0fdd)
ssing something

**Pull request merged**